uiowa/uiowa
# Hotfix: Fix layout builder diff plugin

Resolves #4620 



# How to test


`git checkout i4620 && git pull`
`blt ds --site=sandbox.uiowa.edu`
Visit "https://sandbox.uiowa.ddev.site/admin/config/content/diff/fields" and check that the Content Layout Section field has the "Layout Section Field Diff" plugin selected.
![image](https://user-images.githubusercontent.com/49074566/148254711-509229d2-18c8-4d54-8541-934be361ae42.png)
Visit a page revision comparison (such as "https://sandbox.uiowa.ddev.site/node/656/revisions/view/4621/4626/split_fields") and see that changes made in Layout appear in the revision diff.
